tests/interfaces/default_qubit_2_integration/test_torch_default_qubit_2.py

```diff
@@ -207,23 +246,40 @@ def cost(a, b):
             ops = [qml.RY(a, wires=0), qml.RX(b, wires=1), qml.CNOT(wires=[0, 1])]
             m = [qml.expval(qml.PauliZ(0)), qml.expval(qml.PauliY(1))]
             tape = qml.tape.QuantumScript(ops, m, shots=shots)
-            return torch.hstack(execute([tape], device, **execute_kwargs)[0])
+            res = execute([tape], device, **execute_kwargs)[0]
```

Comment:
I am currently obsessed with this syntax, esp for testing because it will fail if `execute` returns more than one thing so we get a free assertion that `len(res) == 1`. thoughts on adopting this more?
```suggestion
            [res] = execute([tape], device, **execute_kwargs)
```

Reply:
That is nice.  I may start doing that myself 👍

Reply:
Cheeky, I like this one as well! :)